tutoraspects/templates/aspects/jobs/init/init-mysql.sh

```diff
@@ -14,8 +14,8 @@ do
 done
 echo "MySQL is up and running"
 
-# edx-platform database
-mysql -u {{ MYSQL_ROOT_USERNAME }} --password="{{ MYSQL_ROOT_PASSWORD }}" --host "{{ MYSQL_HOST }}" --port {{ MYSQL_PORT }} -e "CREATE DATABASE IF NOT EXISTS {{ SUPERSET_DB_NAME }};"
+# Superset MySQL database, by default points to the same host as edx-platform
+mysql -u {{ MYSQL_ROOT_USERNAME }} --password="{{ MYSQL_ROOT_PASSWORD }}" --host "{{ MYSQL_HOST }}" --port {{ MYSQL_PORT }} -e "CREATE DATABASE IF NOT EXISTS {{ SUPERSET_DB_NAME }} /*!40100 DEFAULT CHARACTER SET utf8mb4 */;"
```

Comment:
The comment will have no effect over the charset:

```suggestion
mysql -u {{ MYSQL_ROOT_USERNAME }} --password="{{ MYSQL_ROOT_PASSWORD }}" --host "{{ MYSQL_HOST }}" --port {{ MYSQL_PORT }} -e "CREATE DATABASE IF NOT EXISTS {{ SUPERSET_DB_NAME }} CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci;;"
```

Reply:
@Ian2012 the comment is actually a MySQL directive that tells it to execute the command only on versions >= 4.1.0 (where these charsets were introduced, I believe). It should work correctly on any modern MySQL. `utf8mb4_0900_ai_ci` is the default collation for `utf8mb4` so I don't think adding that will change anything.